maquetado/scss/styles.scss

```diff
@@ -1,7 +1,8 @@
 @import 'titles';
 @import 'variables';
 @import 'tech';
+@import 'layout';
 
 body {
-  font-family: $font-family;
+	font-family: $font-family;
```

Comment:
Remember the identation

Reply:
this is already solved and will merge when the PR gets accepted